inst/app/modules/main_map.R

```diff
@@ -0,0 +1,51 @@
+
+hk_accidents_valid <- dplyr::filter(hk_accidents, !is.na(latitude) & !is.na(longitude))
```

Comment:
This is interesting. You probably are aware that we sometimes need to prepend the package name to the function because we can have functions named the same across packages. In these scenarios, R will take the function from the latest package loaded. The `filter` function is a particular function which is shared between the `stats` package and the `dplyr` package. I think it is safe to omit the `dplyr::` here though since you `dplyr` is called after the base `stats` package.
```suggestion
hk_accidents_valid <- filter(hk_accidents, !is.na(latitude) & !is.na(longitude))
```

Reply:
hey @avisionh - apart from style consistency, is there another reason for preferring using the order of package loads over the package `::` prepend? I acknowledge that it is not ideal to not have prepends consistently present (or absent), but wouldn't this be a practical compromise given that the package ordering is so implicit and that avoiding namespace conflict is so critical for the code to run?

Reply:
Adding `dplyr::` before `filter` is merely my habit. I messed up once when I first learnt tidyverse a long time ago, so I add `dplyr::` after that to avoid any possible issues. 🤦

Probably could remove the namespace if it is redundant. `dplyr` is common enough that it is "*super obvious*" that we are referring to the filter in dplyr.

Reply:
Agree with both points and would lean towards @KHwong12's comment that `dplyr` is common enough to be obvious that we are referring to the `filter` in there.

The overriding preference for me would be to preprend the package name to `::` but in this instance, the use of `dplyr::filter` (including the overriding of the default `stats::filter` with `dplyr::filter`) is so widespread that most people can take it as given.

As an aside, we could prepend all `dplyr` verbs with `dplyr::` but we can't prepend the `%>%` with `dplyr::`, so I have no clue how to deal with that.

Reply:
I believe the pipe operator is in `margittr` package instead of `dplyr`? Still, it seems that we can't do `magrittr::%>%` (https://stackoverflow.com/questions/27947344/r-use-magrittr-pipe-operator-in-self-written-package). 🤔

Reply:
> I believe the pipe operator is in `margittr` package instead of `dplyr`? Still, it seems that we can't do `magrittr::%>%` (https://stackoverflow.com/questions/27947344/r-use-magrittr-pipe-operator-in-self-written-package). 🤔

They're in both I recall! But for pipes especially adding `::` like totally defeats the point of making things more readable - so we have to be practical as well.